### DISCUSSION TITLE
Detection Model Training/Finetuning

### DISCUSSION THREAD

**Discussion by dmirica94** [mindee/doctr#882] [Q&A]
Hello,
I want to finetune the detection model, but I don't really know what parameters should I adjust:
Any recommendations?

My training dataset: 100k docs.
Any tips for the starting learning rate or how many epochs should I set? 
Also, is there any place where can I see what hyperparameters were used for the released checkpoint?
Thanks for your time and effort! Amazing work, I'm glad that I found this repo :D
